compiler+runtime/src/cpp/jank/runtime/obj/tagged_literal.cpp

```diff
@@ -0,0 +1,99 @@
+#include <jank/runtime/obj/symbol.hpp>
+#include <jank/runtime/core/to_string.hpp>
+#include <jank/runtime/visit.hpp>
+#include <jank/runtime/context.hpp>
+#include <jank/runtime/obj/tagged_literal.hpp>
```

Comment:
`visit.hpp` is a *very* heavy include. You're not using it here, either. Please trim these down to what you're using. Also put `tagged_literal.hpp` first, unless you have a good reason not to.